### DISCUSSION TITLE
Need Help with Deploying pocketbase on Azure

### DISCUSSION THREAD

**Discussion by devloper16** [pocketbase/pocketbase#4307] [Q&A] [answered]
I am trying to deploy pocketbase on azure vm. When I try to deploy it by running PATH/pocketbase serve --http="IP_ADDRESS:80" I am getting Error: listen tcp IP_ADDRESS:80: bind: cannot assign requested address.

I Checked nothing is running on port 80 using "sudo netstat -tuln | grep ':80'" and nothing came up.
I also checked internal firewall and it was inactive using "sudo ufw status".

The port 80 and 443 are open in nsg from both inbound and outbound traffic

I am fairly new to all this, Any help is appreciated!

**Comment by ganigeorgiev** [comment, marked as answer]
Make sure that:

- Your user is allowed to bind to the 80 port. 
- Your IP is correct and properly referencng your VM (ex. mapped in your `/etc/hosts`). An alternative to specifying a fixed IP could be to use `--http="0.0.0.0:80"` to bind to all available IPv4 interfaces.

**Reply by devloper16** [reply]
The user defiantly have permission to do that. I edited the hosts file and bind it to all IPv4's and it is working perfectly (Both api and admin ui). Thanks for you help and quick response.
